filestore/handlers_base.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import six
+import logging
+
+
+class HandlerBase(object):
+    """
+    Base-class for Handlers to provide the boiler plate to
+    make them usable in context managers by provding stubs of
+    ``__enter__``, ``__exit__`` and ``close``
+    """
+    specs = set()
```

Comment:
This is following the fileformat pattern in pims, instance methods should not be setting this and sub-classes should be doing something like

```python

class C(A):
    specs = super.specs()  | set('new', 'ones')
```